extensions/exchanges/hitbtc/exchange.js

```diff
@@ -31,15 +31,15 @@ module.exports = function container (get, set, clear) {
     }
     setTimeout(function () {
       exchange[method].apply(exchange, args)
-    }, 20000)
+    }, 5000)
   }
 
   var orders = {}
 
   var exchange = {
     name: 'hitbtc',
     historyScan: 'forward',
-    makerFee: 0.01,
+    makerFee: -0.01,
```

Comment:
Really?

Reply:
Yessir https://hitbtc.com/fees-and-limits. 0.01% rebate on maker orders